### PR TITLE
[nan-749] default to nested when running nango init

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
When setting up a `nango-integrations` directory using the `nango init` command use the updated nested directory structure. The `nango generate` command will now create files using the nested directory structure. An associated documentation task should be merged before this to reduce confusion. 

## Issue ticket number and link
NAN-749

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/cli/lib/cli.ts [hunk 1]
```diff
@@ -164,7 +165,17 @@ export const generate = async (debug = false, inParentDirectory = false) => {
...
-                fs.writeFileSync(`${dirPrefix}/${name}.ts`, stripped);
+                if (layout_mode === 'root') {
+                    fs.writeFileSync(`${dirPrefix}/${name}.ts`, stripped);
+                } else {
+                    if (!fs.existsSync(`${dirPrefix}/${providerConfigKey}`)) {
+                        fs.mkdirSync(`${dirPrefix}/${providerConfigKey}`);
+                    }
+                    if (!fs.existsSync(`${dirPrefix}/${providerConfigKey}/${type}s`)) {
+                        fs.mkdirSync(`${dirPrefix}/${providerConfigKey}/${type}s`);
+                    }
```
Comment (TBonnin) [inline packages/cli/lib/cli.ts]: I think `mkdirSync` has a `recursive` argument to create a full path without having to create each folder individually

File: packages/cli/lib/services/migration.service.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ async function moveFile(source: string, destination: string, debug = false): Pro
 async function updateModelImport(filePath: string, debug = false): Promise<void> {
     try {
         const data = await fs.promises.readFile(filePath, 'utf8');
-        const updatedData = data.replace(/from '\.\/models/, "from '../../models");
+        const updatedData = data.replace(/from (["'])\.\/models/g, 'from $1../../models');
```
Comment (TBonnin) [inline packages/cli/lib/services/migration.service.ts]: why not simply `data.replace(/\.\/models/g, '../../models');` so we don't have to worry about what kind of quotes are being used?

**Review by TBonnin — APPROVED** [review]

File: packages/cli/lib/cli.ts (hunk 1 above)
Comment (khaliqgant) [reply packages/cli/lib/cli.ts]: Thanks, updated

File: packages/cli/lib/services/migration.service.ts (hunk 2 above)
Comment (khaliqgant) [reply packages/cli/lib/services/migration.service.ts]: Good call, updated, thanks!

**Pull request merged**